src/reference/modules/installation/pages/_partials/debian_repo_new.adoc

```diff
@@ -0,0 +1,39 @@
+Add Rudder's package repository:
```

Comment:
why do you create a new file rather than replacing the previous content ?

Reply:
because we need to keep the doc for old systems for agents

Reply:
ha, you are totally right